File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListBox/PopulatingData/CollectionDataSourcePage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.ListBox.PopulatingData
{
    public partial class CollectionDataSourcePage : UserControl
    {
        public CollectionDataSourcePage()
        {
            InitializeComponent();
            //Set collection of customer as DataSource.
            this.mobjListBox.DataSource = global::Gizmox.WebGUI.Forms.CompanionKit.Controls.Util.CustomerData.GetCustomers();
            //Fill Display and Value Members
            mobjDisplayText.Text = mobjListBox.DisplayMember;
            mobjValueText.Text = mobjListBox.ValueMember;
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListBox/PopulatingData/CollectionDataSourcePage.vb
Namespace CompanionKit.Controls.ListBox.PopulatingData

	Public Class CollectionDataSourcePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()
			'Set collection of customer as DataSource.
            Me.mobjListBox.DataSource = Global.CompanionKit.Controls.Utils.CustomerData.GetCustomers
            'Fill Display and Value Members
            mobjDisplayText.Text = mobjListBox.DisplayMember
            mobjValueText.Text = mobjListBox.ValueMember
		End Sub

	End Class

End Namespace